asciidoc/__metadata__.py

```diff
@@ -1,5 +1,5 @@
 """Module containing metadata about asciidoc."""
 
-VERSION = (10, 0, 2)
+VERSION = (10, 0, 3)
```

Comment:
please don't bump this in a PR. I will bump it once I'm ready to cut a release, update the changelog, etc.

Reply:
OK, sorry, I wasn't sure about that.  Should I redo the PR or will you do that at merge?

Reply:
I'll handle it.